kleinanzeigen_bot/__init__.py

```diff
@@ -571,8 +571,9 @@ def publish_ad(self, ad_file:str, ad_cfg: dict[str, Any], ad_cfg_orig: dict[str,
         utils.save_dict(ad_file, ad_cfg_orig)
 
     def __set_category(self, ad_file:str, ad_cfg: dict[str, Any]):
-        # trigger and wait for automatic category detection
-        self.web_click(By.ID, "pstad-price")
+        # click on something to trigger automatic category detection
+        self.web_click(By.ID, "pstad-descrptn")
+        
```

Comment:
Can you please remove the whitespaces in this line, the linter fails with:
```
Using config file /Users/runner/work/kleinanzeigen-bot/kleinanzeigen-bot/pyproject.toml
************* Module kleinanzeigen_bot
kleinanzeigen_bot/__init__.py:576:0: C0303: Trailing whitespace (trailing-whitespace)

-----------------------------------
Your code has been rated at 9.99/10
```

Reply:
Didn't noticed this at first. I resolved the warning.